.github/workflows/terraform-test.yaml

```diff
@@ -20,12 +20,24 @@ jobs:
           - modules/dashboard
           - modules/eventbridge
           - modules/sns_to_lambda
-     
+          - modules/alerts
+
     steps:
     - uses: dasmeta/reusable-actions-workflows/terraform-test@main
       with:
         aws-region: ${{ secrets.AWS_REGION}}
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         path: ${{ matrix.path }}
-      continue-on-error: true
+    - uses: ravsamhq/notify-slack-action@v2
```

Comment:
do we need to send slack notify on test failure?

Reply:
nope it was for testing and you say we didn't need it to be here. Now i will delete this part.
But you must review the stale.yaml file